src/components/TodoList.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import {List, Loader} from "@mantine/core";
+import {useGetTodosQuery} from "../redux/apiSlice";
+
+
+export const TodoList = (): JSX.Element => {
+
+    const {data = [], isLoading, isSuccess, isError, error} = useGetTodosQuery();
+    let content: JSX.Element | undefined;
```

Comment:
Když už to napíšeš takto přes `let` tak bych tomu nedával, že to může být `undefined`, protože react `undefined` nedokáže vykreslit. Jestli se nepletu tak JSX.Element může být null, takže bych na začátku inicializoval proměnou jako `null` ... je to ale drobnost